backend/spec/requests/books_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+require './spec/helpers/api_helpers'
+
+RSpec.configure do |config|
+  config.include ApiHelpers, type: :request
+end
+
+RSpec.describe "Books", type: :request do
+  describe "POST /books/" do
```

Comment:
Another test for `failure` scenario pls. Test if:
- The expected error status is returned. Usually 422
- The expected error body structure is returned